fastly/validators.go

```diff
@@ -74,3 +75,24 @@ func validateSnippetType() schema.SchemaValidateFunc {
 		"none",
 	}, false)
 }
+
+func validateDictionaryItems() schema.SchemaValidateFunc {
+	max := 10000
```

Comment:
Can we store this is a package level constant like we did with `gofastly.BatchModifyMaximumOperations`.

Reply:
So we could place it as a value in the go-fastly client package.  This way at least it is documented more towards the API level.  It can then be consumed at the Terraform provider validation level.

I can put in two const.  The first will represent diction items and the second will represent acl entries.  Thinking at these two values can be set independently.

Reply:
Sounds sane to me! :+1:

Reply:
The validation function now uses the planned constant from the go-fastly client v1.2.1